src/components/staking/PageValidators.vue

```diff
@@ -1,9 +1,6 @@
 <template>
   <TmPage
     :managed="true"
-    :loading="$apollo.queries.validators.loading"
-    :loaded="!$apollo.queries.validators.loading"
-    :error="$apollo.queries.validators.error"
```

Comment:
Note: `$apollo.queries.validators.error` actually doesn't exist. This needs to be extracted from the result.

```
result(data) {
  this.error = data.error
}

Reply:
You're right. Not sure where I got that from in the docs.

True, either that or an error method.